### DISCUSSION TITLE
Project Control Center is not self-service

### DISCUSSION THREAD

**Discussion by sbtaylor15** [cdfoundation/wg-lfx-requirements#7] [General]
Projects cannot setup or configure Project Control Center.  This has to be done by LF staff.

**Comment by sbtaylor15** [comment]
What is the process to get items fixed, support issue with LFX?

**Comment by TracyRagan** [comment]
Can we have more control so we can add for example links to the project GitHub page. Can LFX tools be enhanced to allow projects to update their own information.

**Comment by TracyRagan** [comment]
Maybe we could have the owner of the GitHub Org to be the owner of the Project Central.
